lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -2349,28 +2436,132 @@ void HdVP2Material::_UpdateShaderInstance(const HdMaterialNetwork& mat)
 
 /*! \brief  Acquires a texture for the given image path.
  */
-const HdVP2TextureInfo&
-HdVP2Material::_AcquireTexture(const std::string& path, const HdMaterialNode& node)
+const HdVP2TextureInfo& HdVP2Material::_AcquireTexture(
+    HdSceneDelegate*      sceneDelegate,
+    const std::string&    path,
+    const HdMaterialNode& node)
 {
     const auto it = _textureMap.find(path);
```

Comment:
This is a local cache of the textures used by this material. If the same texture is used my the material twice the async code will load it twice, and overwrite the first load in _textureMap. I'm not sure if that'll just hurt performance, or if overwriting the original texture info will cause a stability problem.

Reply:
I have added another check to avoid overwriting the texture:
https://github.com/Autodesk/maya-usd/pull/1980/files#diff-95070490bea9fe284d8a3c42c85ed209a5341007e11b85dac96e316dcec6e269R2584-R2588